forte/casscf/casscf_orb_grad.cc

```diff
@@ -232,6 +232,16 @@ void CASSCF_ORB_GRAD::read_options() {
                 zero_rots_[irrep][nu].emplace(p);
                 zero_rots_[irrep][p].emplace(nu);
             }
+
+            // enable rotation between the orbitals within CASSCF_ACTIVE_FROZEN_ORBITAL
+            for (size_t j = 0; j < i; ++j) {
```

Comment:
Should we add an option for this type of behavior? I am OK to take this as the default but we need to document this in the MCSCF documentation.